frontend/public/style/_overrides.scss

```diff
@@ -243,6 +243,14 @@ form.pf-v6-c-form {
   }
 }
 
+.pf-v6-c-hint{
+  margin: var(--pf-t--global--spacer--sm);
```

Comment:
Wondering if we shouldn't adjust margins using PatternFly utility classes on a case by case basis so as not to impact all instances.